### PR TITLE
HMAN-368 temporary PR to test def file verion [DO NOT MERGE] 

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [HMAN-368](https://tools.hmcts.net/jira/browse/HMAN-368) _"Ccd Def store definition changes"_

### Change description ###

Temporary PR to test hmcts/ccd-test-definitions#53 def file changes and hmcts/befta-fw#190.

> NB: BEFTA FW & Def file FTA+ES test PRs are:
> * hmcts/ccd-data-store-api#2086
> * hmcts/ccd-definition-store-api#1279
> * hmcts/ccd-user-profile-api#497

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/ccd-definition-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -23,9 +23,11 @@ java:
     # enable whenever required and provide host url to match with corresponding data-store-api
     ELASTIC_SEARCH_ENABLED: true
 
-    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-2086-es-master
```
Comment (mattnayler) [inline charts/ccd-definition-store-api/values.preview.template.yaml:27]: This should be reveresed if final test-def version changes are merged to `master`.

File: charts/ccd-definition-store-api/values.preview.template.yaml [hunk 2]
```diff
@@ -23,9 +23,11 @@ java:
...
 
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399-java
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399-java
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-497-java
```
Comment (mattnayler) [inline charts/ccd-definition-store-api/values.preview.template.yaml:30]: This should be reveresed if final test-def version changes are merged to `master`.

File: build.gradle [hunk 3]
```diff
@@ -284,8 +284,8 @@ subprojects { subproject ->
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3'
-        testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.5_HMAN-368'
+        testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6-HMAN-368'
```
Comment (mattnayler) [inline build.gradle:288]: These should be updated to match final version numbers if being merged to `master`.

**Comment by marianadpereira** [comment]
closing as this is not needed anymore.

**Pull request closed without merging**